monitoring/error500.js

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+/* eslint-disable */
```

Comment:
`eslint` is your friend, don't disable `eslint` globally. you can disable specific rules as needed (e.g. no-console).

BTW: `eslint` would had warned you regarding `body.results != []`

Reply:
`/* eslint-disable no-console */` is OK though, since this will populate the script log.